<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscriberMainWindow</class>
 <widget class="QMainWindow" name="TranscriberMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TranscriberMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widgetUpperPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <widget class="QPushButton" name="pushButtonLoadFileName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditFileName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>391</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>file-name.wav</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCurDocPosX">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCurSampleInd">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>627</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>DocPosX</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>618</x>
         <y>37</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SampleInd</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPlay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPause">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditRecognizerName">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>0</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>recog name</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>0</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Marker Level</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="radioButtonPhoneLevel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>26</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonWordLevel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>11</y>
          <width>49</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Word</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEditMarkerText">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>15</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>annotation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>-2</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Marker ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMarkerId">
       <property name="geometry">
        <rect>
         <x>896</x>
         <y>-2</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>###</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSaveAudioAnnot">
       <property name="geometry">
        <rect>
         <x>486</x>
         <y>-1</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxCurMarkerStopOnPlayback">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>42</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Stops Playback</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>638</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cursor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSegmentComposerPlay">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>SegPlay</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetSamplesMain" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollBar" name="horizontalScrollBarSamples">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AudioSamplesWidget" name="widgetSamples" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>widgetSamples</zorder>
      <zorder>horizontalScrollBarSamples</zorder>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetBottomPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEditLogger">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;a&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;b&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;c&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditAudioSegmentsComposer"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AudioSamplesWidget</class>
   <extends>QWidget</extends>
   <header>AudioSamplesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
